Sources/ProjectDescription/TargetAction.swift

```diff
@@ -10,20 +10,62 @@ public struct TargetAction: Codable, Equatable {
         case post
     }
 
+    /// How to execute the target action
```

Comment:
Should not most of the changes here be only in `TuistCore.TargetAction`? 🙂

Reply:
Good call! I've refactored that a bit.